app/js/handlebarsScoreTable.ts

```diff
@@ -411,8 +411,20 @@ function addEventListenersForMoreInfoButtons() {
             document.querySelector('tr[data-id="' + userId + '"]').classList.toggle('hide')
             if(this.textContent == 'More info') {
                 this.textContent = 'Less info';
+                e.target.parentElement.parentElement.style.background = "#fff";
```

Comment:
instead of dynamically changing the style attribute, it would be preferable to add a css class which applies this style. this is better because that way separation of concerns is maintained - e.g. the scss files contain all the css styling, there is no "hidden styling" somewhere in the middle of the js

Reply:
fixed!